Description: ensure that rst-buildhtml progress messages end with newlines
Author: Jakub Wilk <jwilk@debian.org>
Bug: http://sourceforge.net/tracker/?group_id=38414&atid=422030
Last-Update: 2012-12-30

--- a/tools/buildhtml.py
+++ b/tools/buildhtml.py
@@ -206,7 +206,7 @@
             del subdirectories[:]
             return
         if not self.initial_settings.silent:
-            errout.write('/// Processing directory: %s' % directory)
+            errout.write('/// Processing directory: %s\n' % directory)
             sys.stderr.flush()
         # settings.ignore grows many duplicate entries as we recurse
         # if we add patterns in config files or on the command line.
